packages/crud/src/crud/useRouterQueryFilters.ts

```diff
@@ -48,13 +48,14 @@ const useRouterQueryFilters = (args) => {
 
         // Set relationalObjectValue to render the correct title
         const { pk = 'title' } = filterField.module
+        const { optionLabelKey = 'title' } = filterField
         const relationalObjectValue =
           typeof relationalObject === 'object'
-            ? relationalObject[pk]
+            ? relationalObject[optionLabelKey] ?? relationalObject[pk]
             : relationalObject
         return {
           ...defaultReturn,
-          [relationalObjectKey]: relationalObjectValue,
+          [relationalObjectKey]: relationalObjectValue
```

Comment:
We should have commas here. This shouldn't be happening! Linter is acting weird in this pr